src/core/sys/posix/iconv.d

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+
+    D binding for the POSIX iconv library.
+
+    See_Also:
+       http://pubs.opengroup.org/onlinepubs/009695399/functions/iconv_open.html
+
```

Comment:
It looks like it indeed should have a header, thanks! I've updated this PR with the hearer like in `mqueue.d` module.